src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -1,16 +1,14 @@
 package sbtprotoc
 
-// avoid ambiguous import when sjsonnew.CollectionFormats.seqFormat is brought in scope by ignoring sbt 1.3 implicit
-// https://github.com/sbt/sbt/commit/33194233698ab49682d89091a332edb808cb75bc#diff-cafb18f4282f4fff753f9efb35169d46R39
-import sbt.{fileJsonFormatter => _, _}
```

Comment:
`sbt.fileJsonFormatter` was introduced in sbt 1.3, so the wildcard is now safe since we built against a previous version